src/subcommand/wallet/list.rs

```diff
@@ -0,0 +1,92 @@
+use {super::*, bitcoincore_rpc::RpcApi};
+
+pub(crate) fn run(options: Options) -> Result {
+  let index = Index::open(&options)?;
+  index.index()?;
+
+  let client = options.rpc_client()?;
+
+  let unspent = client.list_unspent(None, None, None, None, None)?;
+
+  let mut utxos = Vec::new();
+  for utxo in unspent {
+    let output = OutPoint::new(utxo.txid, utxo.vout);
+    match index.list(output)? {
+      Some(List::Unspent(ordinal_ranges)) => {
+        utxos.push((output, ordinal_ranges));
+      }
+      Some(List::Spent) => {
+        bail!("Output {output} in wallet but is spent according to index")
+      }
+      None => bail!("Ordinals index has not seen {output}"),
+    }
+  }
+
+  for (start, size, output, rarity, name) in list(utxos) {
+    println!("{start}\t{size}\t{output}\t{rarity}\t{name}");
```

Comment:
I was thinking that maybe we should have more consistent ordering of `ord wallet identify` and `ord wallet list`.

For `identify`: `output ordinal offset rarity name`
And `list`: `output ordinal size rarity name`

Since `list` should be in order, this is also pretty natural, as all the ranges in one output will be grouped together.